### DISCUSSION TITLE
Why is it recommended to also use the apache2/nginx ppa when using the php ppa?

### DISCUSSION THREAD

**Discussion by enbrnz** [oerdnj/deb.sury.org#1753] [General]
In the discription of [the PHP PPA](https://launchpad.net/~ondrej/+archive/ubuntu/php/) I see the following caveat:
> [...]
> 2. If you are using apache2, you are advised to add ppa:ondrej/apache2
> 3. If you are using nginx, you are advised to add ppa:ondrej/nginx-mainline
   or ppa:ondrej/nginx

In the description of the NGINX PPAs, there is the following description:
> This branch follows latest NGINX (Mainline|Stable) packages compiled against latest OpenSSL for HTTP/2 and TLS 1.3 support.

And finally in the description of the Apache2 PPA:
> This branch follows latest Apache2 packages as maintained by the Debian Apache2 team with couple of compatibility patches on top.

This sounds like that additional features are provided by providing more up to date versions of the web servers. But I would like to understand if there are certain incompatibilities between the PHP packages and Ubuntu distribution apache2/nginx packages.

I have read the FAQ and had a quick search in the issue tracker, but haven't been able to find anything yet.

Additionally, there are no nginx/apache2 packages for Jammy yet. Are they just not necessary as Jammy's Apache/Nginx versions are more up to date, or will they come later?

**Comment by oerdnj** [comment]
There were some incompatibilities in the past, but there are none as far as I am aware.

Jammy hasn't been released yet.
